tests/integration/tier0/basic-sanity-checks/test_basic_sanity_checks.py

```diff
@@ -107,7 +105,7 @@ def test_c2r_latest_older_unsupported_version(convert2rhel):
     os.environ["CONVERT2RHEL_ALLOW_OLDER_VERSION"] = "1"
 
     with convert2rhel(f"--no-rpm-va --debug") as c2r:
-        if "centos-6" in booted_os or "oracle-6" in booted_os:
+        if SYSTEM_RELEASE_ENV in ("centos-6", "oracle-6"):
             assert c2r.expect("You are currently running 0.01", timeout=300) == 0
             assert c2r.expect("Only the latest version is supported for conversion.", timeout=300) == 0
```

Comment:
Not sure if we still need this. We will drop support for RHEL6 in v1.2, our CI does not even run those machines anymore.

Reply:
Yep, it was there just for the sake of potential manual testing.
If that's okay with you, I would rather merge this one, and remove the `rhel6` like distros in a different PR.
I would say #723 as it's the one, I'm working on now.

Reply:
Totally fine with it! Thanks for explaining.